kube_scheduler/datadog_checks/kube_scheduler/data/conf.yaml.example

```diff
@@ -0,0 +1,29 @@
+init_config:
+
+instances:
+
+  ## @param prometheus_url - string - required
+  ## The URL where your application metrics are exposed by Prometheus.
+  #
+  - prometheus_url: http://localhost:10251/metrics
+
+  ## @param tags - list of key:value elements - optional
+  ## List of tags to attach to every metrics, events and service checks emitted by this integration.
+  ## Learn more about tagging: https://docs.datadoghq.com/tagging/
+  #
+  #  tags:
+  #    - <KEY_1>:<VALUE_1>
+  #    - <KEY_2>:<VALUE_2>
+
+  ## @param leader_election - boolean - default: true
+  ## Monitor the leader-election process on the kube-system:kube-scheduler endpoints
+  #
+  #  leader_election: true
+
+  ## @param prometheus_timeout - integer - optional - default: 10
+  ## Overrides the default timeout value in second
+  #
+  #  prometheus_timeout: 10
+
```

Comment:
there are some trailing empty lines at the end of this file that can be removed :)